docs/SETUP_PROVISIONING.md

```diff
@@ -0,0 +1,1505 @@
+| tls_privkey_pem | staging | private key for the certificate in PEM format |
+| tls_chain_pem | staging | contains the additional intermediate certificate or certificates that web browsers will need in order to validate the server certificate |
+| tls_fullchain_pem | live | all certificates, including server certificate (aka leaf certificate or end-entity certificate). The server certificate is the first one in this file, followed by any intermediates. |
+| tls_privkey_pem | live | private key for the certificate in PEM format |
+| tls_chain_pem | live | contains the additional intermediate certificate or certificates that web browsers will need in order to validate the server certificate |
+
+When provisioning a new staging or live environment, ``ops/scripts/setup_cert.sh`` will attempt to pull these variables if they exist.
+This way, we won't unnecessarily creat certificate everytime we need to re-create a staging and deployment, 
+thus reducing the risk of hitting weekly rate-limit for certificate creation imposed by Let's Encrypt.
+
+> Since an elastic IP address is being used, you might need to delete the entry
+in the `~/.ssh/known_hosts` file associated with the elastic IP address if this
+is not the first time you have performed the plays in the playbook.
+
+### Restoration of database backups
+
+```
+#Go to environment directory
+$ cd <path>/gigadb-website/ops/infrastructure/envs/staging
+
+#Terminate existing RDS service
+$ terraform destroy --target module.rds
+
+#Copy override.tf to staging environment
+$ ../../../scripts/tf_init.sh --project gigascience/forks/pli888-gigadb-website --env staging --restore-backup
+
+#Backups can either be restored to its latest restorable time or to a specific
+#time.
+
+#To restore to latest restorable time - need to override database name as this 
+#will come from the backup
+$ terraform apply -var source_dbi_resource_id="db-6GQU4LWFBZI34AOR5BW2MEQFLU" -var gigadb_db_database="" -var use_latest_restorable_time="true"
+
+#To restore to specific time in backup - need to override database name as this 
+#will come from the backup
+$ terraform apply -var source_dbi_resource_id="db-6GQU4LWFBZI34AOR5BW2MEQFLU" -var gigadb_db_database="" -var utc_restore_time="2021-10-27T06:02:12+00:00"
+```
+
+### Troubleshooting
+
+If you meet the error:
+ ```
+ Error: Error launching source instance: MissingInput: No subnets found for the default VPC 'vpc-a717e2ce'. Please specify a subnet.
+ ```
+
+Then running the following command to create a default subnet will fix it:
+
+ ```
+ aws ec2 --profile UserNameOfAdminUser create-default-subnet --region ap-east-1 --availability-zone ap-east-1a
+ ```
+
+All SQL statements executed on the PostgreSQL RDS instance are logged in staging
+deployments of GigaDB. These `postgres.log` files are available from the AWS
+console for the RDS service in the `Log & events` tab.
+
+
+### Tools
```

Comment:
Can probably delete everything from here to the end of this file because this part of the documentation has been covered above?